src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { Errors } from '../../types/ErrorTypes';
+
+type Props = {
+  onAdd: (todo: Todo) => void,
+  userId: number,
+  setErrorMsg: (errorMsg: Errors | null) => void,
+};
+
+export const NewTodo: FC<Props> = (
+  {
+    onAdd, userId, setErrorMsg,
+  },
+) => {
+  const [inputTitle, setInputTitle] = useState('');
```

Comment:
Try to do it by useRef. Now component is rerendering everytime user type singe letter and we dont need that
https://react.dev/reference/react/useRef